uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/broker/BrokerHost.java

```diff
@@ -321,4 +323,19 @@ public void disableMultipleWpj() {
         ThreadUtils.sleepSafely(500, TAG, "Wait before launch.");
         launch();
     }
+
+    /**
+     * Changes flight provider to local flights provider and sets
+     * PRTv3 flight flag to true.
+     */
+    public void enablePrtV3() {
```

Comment:
can we make it generic. 
public void setLocalFlightConfigurations(Map<string, string> flightConfigurations)

Reply:
Added generic method in addtion to this.